libraries/AP_Airspeed/AP_Airspeed_ASP5033.cpp

```diff
@@ -0,0 +1,141 @@
+/*
+   This program is free software: you can redistribute it and/or modify
+   it under the terms of the GNU General Public License as published by
+   the Free Software Foundation, either version 3 of the License, or
+   (at your option) any later version.
+
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU General Public License for more details.
+
+   You should have received a copy of the GNU General Public License
+   along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/*
+  backend driver for airspeed from www.qio-tek.com a I2C ASP5033 sensor
+ */
+
+#include "AP_Airspeed_ASP5033.h"
+#include <GCS_MAVLink/GCS.h>
+#include <AP_Common/AP_Common.h>
+#include <AP_HAL/AP_HAL.h>
+#include <AP_HAL/I2CDevice.h>
+#include <AP_Math/AP_Math.h>
+
+extern const AP_HAL::HAL &hal;
+
+#define ASP5033_I2C_ADDR 0x6D
+#define Sys_config 0xA5
+#define CMD 0x30
+#define CMD_measure 0x0A
+#define Press_data 0x06
+#define Temp_data 0x09
+#define WAI_REG 0x01
+#define DEVICE_ID  0x00
+#define Sensor_ready 0x08
```

Comment:
For a define like this we do the name as all upper case. A couple of these need to be fixed up.

Reply:
fixed